lib/evm_database.rb

```diff
@@ -32,7 +32,11 @@ def self.seedable_model_class_names
   end
 
   def self.seed_primordial
-    seed(PRIMORDIAL_CLASSES)
+    if ENV['SKIP_PRIMORDIAL_SEED'] == 'true' && MiqDatabase.count > 0
+      STDERR.puts "SKIPPING primordial seeding via SKIP_PRIMORDIAL_SEED, unset and re-run if this causes problems."
```

Comment:
Sorry I didn't say this before, but we should probably keep things like this looking fairly similar: https://github.com/ManageIQ/manageiq/blob/855ee77e434c983f23feb33e63499f07d68ddb76/config/initializers/rack_ruby_prof.rb#L3-L4